backend/bib_example_filter_client.py

```diff
@@ -0,0 +1,26 @@
+
+import grpc
```

Comment:
I am a little bit hesitant to commit binary files in git repositories. We could add it via the git-lfs mechanism. I am not 100% if it would be necessary, since in the client I generate entries to the database first, which then can be queried. If one provides a prrepopulated database, one might run into unique-name conflicts. But I have no strong opinion about that. What data would you provide in this initial database ?

Reply:
I mean either a sqlite file without git-lfs as the file will be very small or a pre-init script documented in the readme that populate everything correctly

Reply:
@markdoerr I think this point is still in progress. An other idea would have been to use django fixtures. So you can just dumpdata of your db into json file and add loaddate command in the readme

Reply:
@AMontagu, 
that sounds like a good solution - I do not know, how this will feel in practice, but we can try. I should not be to hard for the users. And in the docker case we can even automate it with a "singleton-like" process (using,, e.g. docker compose profiles).

Reply:
for the other comments, I will try to comment / fix tonight or latest tomorrow evening...